articles/how-to-collect-sensor-data-on-your-iphone-or-android-device-using-matlab/index.md

```diff
@@ -0,0 +1,131 @@
+a new name, you click `ok`. 
+Let us now go back to the command window and begin writing some codes. We will view the image using the `imread` function and then use the `imshow` function 
+to view it. 
+```Matlab
+I = imread('filename.jpg');
+imshow(I)
+```
+![showing our captured image](/howto-collect-sensor-data-on-your-iphone-or-android-device-using-matlab/engineering-education/sensor_four.jpeg)
+After accessing your image, you can do anything you wish with the image. For example, you can change the image to grayscale and many more options. The code is;
+```Matlab
+B = rgb2gray(I);
+imshow(B)
+```
+![the grayscale image](/howto-collect-sensor-data-on-your-iphone-or-android-device-using-matlab/engineering-education/sensor_five.jpeg)
+As you can see, it was easy to use our mobile Matlab to acquire the image and perform some quick image processing. You can write all your commands in a 
+script so that you can use them anytime on your mobile Matlab or desktop, or laptop. First, click on the `+` sign on the file section and click on the 
+`script.`
+This technology makes it possible to use your commands anywhere in the Matlab drive, where you are syncing all your files and data. Once you sync all the
+data into your Matlab drive, you can access them in your Matlab drive. Alternatively, you can access these data online on your Matlab drive. To do this,
+click [here](https://www.mathworks.com/products/matlab-drive.html) and click on `Matlab drive online`.
+![location of Matlab drive online](/howto-collect-sensor-data-on-your-iphone-or-android-device-using-matlab/engineering-education/sensor_six.png)
+When you click on the `Matlab drive online`, provide your information to log into your Mathworks account. When this is successful, you will find all your 
+files here. You can download this data into your pc and use them from there. 
+![structure of the drive](/howto-collect-sensor-data-on-your-iphone-or-android-device-using-matlab/engineering-education/sensor_seven.png)
+
+### Using phone sensors
+These sensors are;
+- Acceleration sensors
+- Angular velocity sensors
+- Orientation sensors
+- Magnetic field sensors
+- Position sensors
+
+To access the sensors, click on the file menu and then sensors. When you open the sensor window, you find the various sensors provided by your phone. You 
+can log data from all these available sensors or just one sensor depending on what you want to do. You can either log data and save it as a `.mat` file or 
+stream data to Matlab. In this case, we will look at how to log data and access the collected data. We are going to use the collected data to calculate the 
+distance traveled. To collect data to calculate the distance, we are going to collect latitudinal and longitudinal information. To do this, you will move 
+with your phone as it collects the data. 
+To set your phone for this, we will activate the position sensor then click on the start button for it to start collecting this data. Once you have done this, 
```

Comment:
```suggestion
To set your phone for this activity, activate the position sensor then click on the start button and it will immediately start collecting this data. 
```